### PR TITLE
machines-types update based on new standard/machine-types/core

### PR DESCRIPTION
**Pull request by jouvin**
- Requires https://github.com/quattor/template-library-standard/pull/140
- Support removed for WN profile cloning

### CODE REVIEW

File: machine-types/grid/condor_server.pan [hunk 1]
```diff
@@ -30,32 +29,32 @@ variable VO_GRIDMAPFILE_MAP_VOMS_ROLES ?= true;
...
-include { if ( NFS_CLIENT_ENABLED ) 'features/nfs/client/config' };
+include if ( NFS_SERVER_ENABLED ) 'features/nfs/server/config';
+include if ( NFS_CLIENT_ENABLED ) 'features/nfs/client/config';
 
 
 
 #
 # Add site specific configuration, if any
-include { return(CONDOR_SERVER_CONFIG_SITE) };
+include return(CONDOR_SERVER_CONFIG_SITE);
```
Comment (Pansanel) [inline machine-types/grid/condor_server.pan:49]: Is the return statement necessary?

File: personality/wn/rpms/config.pan [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
-    #SELF['{atlas-worker-node}'] = nlist();
-    SELF[{'xrootd-client'}]    = nlist();
-    SELF[{'ganglia-gmond'}]    = nlist();
-    SELF[{'condor'}]           = nlist();
-    # Not available on all sites
-#    SELF[{'cvmfs-init-scripts'}] = nlist();
-  };
-  SELF;
+
+    if(OS_VERSION_PARAMS['major'] == 'sl6'){
```
Comment (Pansanel) [inline personality/wn/rpms/config.pan:9]: Spaces are missing before and after '(' and ')' characters. The statement could also be removed, as sl6 is not in production any more.

File: personality/wn/rpms/config.pan [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
+        SELF['{emi-wn}'] = dict();
+    }else{
```
Comment (Pansanel) [inline personality/wn/rpms/config.pan:11]: Missing spaces before and after `else`

File: personality/wn/rpms/config.pan [hunk 4]
```diff
@@ -1,20 +1,22 @@
...
+        SELF['{wn}'] = dict();
+    };
+
+    if (ATLAS_SUPPORT ) {
```
Comment (Pansanel) [inline personality/wn/rpms/config.pan:15]: Missing a space before `ATLAS_SUPPORT`

File: machine-types/grid/wn.pan [hunk 5]
```diff
@@ -1,28 +1,74 @@
...
+
+#
+#lemon monitoring specific for WNs
+#
+variable LEMON_AGENT_NODE_SPECIFIC = if ( LEMON_CONFIGURE_AGENT ) {
+    "monitoring/lemon/client/wn/config";
+} else {
+    null;
 };
+include LEMON_AGENT_NODE_SPECIFIC;
```
Comment (Pansanel) [inline machine-types/grid/wn.pan:68]: I think that lemon support can be safely dropped.

File: machine-types/grid/condor_server.pan (hunk 1 above)
Comment (jouvin) [reply machine-types/grid/condor_server.pan:49]: You are completely right!

File: machine-types/grid/base.pan [hunk 6]
```diff
@@ -1,277 +1,140 @@
...
-                         return("monitoring/lemon/client/base/service");
-                       } else {
-                         return(null);
-                       };
-include { LEMON_AGENT_INCLUDE };
+    "monitoring/lemon/client/base/service";
+} else {
+    null;
+};
+include LEMON_AGENT_INCLUDE;
```
[the remark this answers is not in the document]
Comment (jouvin) [reply machine-types/grid/base.pan:110]: ok

[the remark this answers is not in the document]
Comment (Pansanel) [reply machine-types/grid/base.pan:110]: We could drop all the lemon-related bloc.

File: machine-types/grid/base.pan [hunk 7]
```diff
@@ -1,277 +1,140 @@
...
-
-#
-# profile_base for profile structure
-#
-include { 'quattor/profile_base' };
-
-#
-# NCM core components
-#
-include { 'components/spma/config' };
```
Comment (Pansanel) [inline machine-types/grid/base.pan:48]: This include seems to be replaced by `'components/spma/functions';` in the [core-init.pan](https://github.com/quattor/template-library-standard/blob/master/machine-types/core-init.pan) file. Is that right?

Comment (jrha) [reply machine-types/grid/base.pan:48]: @jouvin?

Comment (jouvin) [reply machine-types/grid/base.pan:48]: Sorry for the lack of follow-up. I'll try do to this today.

Comment (jrha) [reply machine-types/grid/base.pan:48]: Thanks, the rest of this looks good, just need an answer to this question.

Comment (jouvin) [reply machine-types/grid/base.pan:48]: I checked but this line was removed a while ago. For some reason, it seems that GH didn't realize it...

**Review by jrha — APPROVED** [review]
LGTM

**Pull request merged**